### DISCUSSION TITLE
[Android] Shared ViewModel Problem

### DISCUSSION THREAD

**Discussion by Injent** [adrielcafe/voyager#246] [Q&A]
Hello everyone. I ran into the problem of using a shared viewmodel. I have a screen with nested navigation and here is it:
```
object NestedNavigatorScreen : AndroidScreen() {
    @Composable
    override fun Content() {
        Navigator(screen = ScreenA)
    }
}
```
The problem is that I need to use one `ViewModel` instance in two screens using one `ViewModelStore` from `NestedNavigatorScreen`. But since following the documentation each screen has its own `ViewModelStore` I can't create or get the same ViewModel using the `ViewModelStore` from the current screen.
DI I used: Koin

**IN SHORT**: how to make shared viewmodel with common ViewModelStoreOwner from NestedNavigatorScreen
